packages/transformers/sass/src/SassTransformer.js

```diff
@@ -83,6 +94,17 @@ export default new Transformer({
           asset.addIncludedFile({filePath: included});
         }
       }
+
+      if (result.map != null) {
+        let map = new SourceMap();
+        let {mappings, sources, names} = JSON.parse(result.map);
+        map.addRawMappings(
+          mappings,
+          sources, // .map(s => path.relative(options.projectRoot, s)),
```

Comment:
Should that be commented back in?

Reply:
nope, was just copy-paste from less but apparently sass already does relative mapping

Reply:
How does it know it should be relative to the projectRoot?

Reply:
> `OutFile`: This string option is the location that Sass expects the generated CSS to be saved to. It’s used to determine the URL used to link from the generated CSS to the source map, and from the source map to the Sass source files.

It's due to `outFile` will remove it and see if that outputs the full paths